encryption.tex

```diff
@@ -1,13 +1,30 @@
-\newpage
-\section{Client Encryption}
+\section{System Disk Encryption}
+
+An encrypted system disk, prevents that the data contained in it can be clone or replicated without the passphrase or authentication server. For this design, the disks will be encrypted through kickstart passphrase and then removed once the remote Tang server are reached, which means, of non-authorized user gains physical access to the server:
```

Comment:
it can be "cloned"...through "a" kickstart.......server "is" reached.....

I don't understand the idea of the last phrase , "of non-authorized....."

Reply:
fixed; also, it was "if", not "of"